lib/web/JsonpChunkLoadingRuntimeModule.js

```diff
@@ -18,6 +18,7 @@ class JsonpChunkLoadingRuntimeModule extends RuntimeModule {
 		this.jsonpScript = jsonpScript;
 		this.linkPreload = linkPreload;
 		this.linkPrefetch = linkPrefetch;
+		this.runtimeRequirements.add(RuntimeGlobals.hasOwnProperty);
```

Comment:
Don't add it here, but in JsonpTemplatePlugin where this runtime module is added